src/toplevelprops.ts

```diff
@@ -31,6 +33,12 @@ export interface TopLevelProperties {
    * __Default value__: `pad`
    */
   autosize?: AutosizeType | AutoSizeParams;
+
+  /**
+   * A global data store for named datasets. This is a mapping from names to inline datasets.
+   * This can be an array of objects or primitive values or a string. Arrays of primitive values are ingested as objects with a `data` property.
+   */
+  datasets?: Dict<InlineDataset>;
```

Comment:
GIve this a name `Datasets`  so it shows up nicer in JSON schema?

Reply:
Done